Partition (WBP)/Functions/VI0036C.VTI0036.fun.yml

```diff
@@ -0,0 +1,1134 @@
+    
+    * Unique Key Data.
+    Def_List Name(#wLKData) Fields(#VTF_KEY1 #VTF_KEY2 #VTF_KEY3) Type(*WORKING) Entrys(1)
+    
+    * Return messages.
+    Def_List Name(#wLRet) Fields(#BIFMSGC #BIFMSG1 #BIFMSG2) Counter(#BIFMSGCNT) Type(*WORKING) Entrys(*MAX)
+    
+    Define Field(#LF_rc) Reffld(#BIFA002A)
+    Define Field(#LF_DB_Type) Type(*CHAR) Length(32)
+    Define Field(#LF_i) Reffld(#Std_Count)
+    Define Field(#LF_Original_Count) Reffld(#Std_Count)
+    Define Field(#LF_DB_Count) Reffld(#Std_Count)
+    Define Field(#LF_DataShown) Reffld(#BIFB001B)
+    
+    * Local Copies of Table Columns - matches following List fields
+    Define Field(#LF_V_ID) Reffld(#V_ID)
+    Define Field(#LF_V_NAME) Reffld(#V_NAME)
+    Define Field(#LF_V_SURNAME) Reffld(#V_SURNAME)
+    Define Field(#LF_V_LASTNAME) Reffld(#V_LASTNAME)
+    Define Field(#LF_V_DATE) Reffld(#V_DATE)
+    Define Field(#LF_V_SALARY) Reffld(#V_SALARY)
+    Define Field(#LF_V_ACTIVE) Reffld(#V_ACTIVE)
+    Define Field(#LF_V_DESC) Reffld(#V_DESC)
+    Define Field(#LF_V_TAG) Reffld(#V_TAG)
+    Define Field(#LF_V_TIME) Reffld(#V_TIME)
+    Define Field(#LF_V_RAWDATA) Reffld(#V_RAWDATA)
+    Define Field(#LF_V_DATA) Reffld(#V_DATA)
+    Define Field(#LF_V_CODE) Reffld(#V_CODE)
+    Define Field(#LF_V_xml) Reffld(#V_xml)
+    
+    
+    * Local Copies of Table Columns - matches following List fields
+    Define Field(#LF_MYS_ID) Reffld(#V_MYS_ID)
+    Define Field(#LF_MYS_NAME) Reffld(#V_MYS_NAME)
+    Define Field(#LF_MYS_SURNAME) Reffld(#V_MYS_SURNAME)
+    Define Field(#LF_MYS_LASTNAME) Reffld(#V_MYS_LASTNAME)
+    Define Field(#LF_MYS_DATE) Reffld(#V_MYS_DATE)
+    Define Field(#LF_MYS_SALARY) Reffld(#V_MYS_SALARY)
+    Define Field(#LF_MYS_ACTIVE) Reffld(#V_MYS_ACTIVE)
+    Define Field(#LF_MYS_DESC) Reffld(#V_MYS_DESC)
+    Define Field(#LF_MYS_TAG) Reffld(#V_MYS_TAG)
+    Define Field(#LF_MYS_TIME) Reffld(#V_MYS_TIME)
+    Define Field(#LF_MYS_RAWDATA) Reffld(#V_MYS_RAWDATA)
+    Define Field(#LF_MYS_DATA) Reffld(#V_MYS_DATA)
+    Define Field(#LF_MYS_CODE) Reffld(#V_MYS_CODE)
+    Define Field(#LF_MYS_xml) Reffld(#V_MYS_xml)
+    
+    * #XG_ALL includes all Fields in the Table
+    Group_By Name(#XG_ALL) Fields(#V_ID #V_NAME #V_SURNAME #V_LASTNAME #V_ACTIVE #V_CODE #V_DATE #V_DESC #V_DATA #V_RAWDATA #V_SALARY #V_TAG #V_TIME #V_XML)
+    Def_List Name(#w_lOriginal) Fields(#XG_ALL) Counter(#LF_Original_Count) Type(*WORKING)
+    
+    Group_By Name(#XG_MysqlALL) Fields(#V_MYS_ID #V_MYS_NAME #V_MYS_SURNAME #V_MYS_LASTNAME #V_MYS_ACTIVE #V_MYS_CODE #V_MYS_DATE #V_MYS_DESC #V_MYS_DATA #V_MYS_RAWDATA #V_MYS_SALARY #V_MYS_TAG #V_MYS_TIME #V_MYS_XML)
+    Def_List Name(#w_lMysqlOriginal) Fields(#XG_MysqlALL) Counter(#LF_Original_Count) Type(*WORKING)
+    
+    Def_List Name(#w_lDB) Fields(#XG_ALL) Counter(#LF_DB_Count) Type(*WORKING)
+    
+    Use Builtin(Get_Session_Value) With_Args(DBUT) To_Get(#LF_DB_Type #LF_rc)
+    
+    #Com_Owner.mthGetExecutionType()
```

Comment:
This method call must be AFTER
```
Clr_List Named(#wLRet)
```
`#wLRet` is the message list. We need to see the messages issued by the method.

Reply:
Done